stylesheets/components/Inputs/Input.css

```diff
@@ -0,0 +1,177 @@
+
+
+/* Input Field */
+.input--field {
+  height: 56px;
+  width: 344px;
+  padding: 12px;
+  border-radius: 8px;
+  border: 3px solid var(--component-grey-01--hover);
+  background-color: hsla(0, 0%, 88%, 50%);
+  color: var(--component-grey-05);
+}
+
+.input--field::placeholder {
+  color: var(--component-grey-03);
+}
+
+.input__container {
+  position: relative;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* Input Sizes */
+/* Keep the width fixed */
+.input--field--small {
+  width: 240px;
+}
+
+/* Keep the width fixed */
+.input--field--large {
+  width: 496px;
+}
+
+/* Outline Input Styling */
+
+.input--field--outline {
+  background-color: var(--white);
+  border: 3px solid var(--component-grey-05);
+  color: black;
+}
+
+.input--field--outline:focus {
+  border: 3px solid var(--component-grey-03);
+}
+
+.input--field--outline::placeholder {
+  color: black;
+}
+
+.input--icon {
+  position: absolute;
+  display: block;
+  right: 12px;
+  outline: 0px;
+}
+
+/* Input Label Styling */
+.input--sidelabel {
+  color: var(--component-grey-06);
+  height: 48px;
+  width: fit-content;
+  text-align: right;
+  margin-right: 1rem;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+
+/* Input Top Label */
+
+.input--toplabel {
+  background-color: var(--white);
+  position: absolute;
+  border-radius: 4px;
+  top: -8px;
+  left: 8px;
+  padding: 2px 4px;
+  margin: 0px 4px;
+  font-size: 12px;
+  font-weight: 500;
+}
+
+
+/* Input Status Messages */
+.input--msg {
+  position: absolute;
+  bottom: -30px;
+  left: 12px;
+}
+
+.--warning {
+  color: var(--failure-color);
+}
+
+.--success {
+  color: var(--success-color);
+}
+
+.--primary {
+  color: var(--component-blue-05);
+}
```

Comment:
If these are general classes then you shouldn't start the name with double hyphens, it looks like a variable, you can name them like `.text--warning, .text--success` etc. and if they are specific to the inputs, then you can add the `.input` prefix to them

Reply:
Have corrected it in the latest commit. Thank You .